packages/text-document-utils/test/position.test.ts

```diff
@@ -0,0 +1,154 @@
+import {
+    isBefore,
+    positionContained,
+    rangeContained,
+    getIndentLevel,
+    positionContainedStrict,
+    indent,
+    positionAt
+} from '../src/position';
+import { printOptions } from '../src/text-formatting';
+import { Position } from 'vscode-languageserver-types';
+
+// Mocking Position.create
+jest.mock('vscode-languageserver-types', () => ({
```

Comment:
Why do we need the mock here?

Reply:
remove the mocking at all, factory functions being directly used.